client/my-sites/themes/themes-selection.jsx

```diff
@@ -247,6 +250,43 @@ class ThemesSelection extends Component {
 		return options;
 	};
 
+	getActionLabel = ( theme ) => {
+		return this.props.getActionLabel( theme );
+	};
+
+	getPremiumThemePrice = ( theme ) => {
+		return this.props.getPremiumThemePrice( theme );
+	};
+
+	getScreenshotUrl = ( theme, themeOptions ) => {
+		return this.props.getScreenshotUrl( theme, themeOptions );
+	};
+
+	isThemeActive = ( theme ) => {
+		return this.props.isThemeActive( theme );
+	};
+
+	isInstallingTheme = ( theme ) => {
+		return this.props.isInstallingTheme( theme );
+	};
+
+	getCarouselBlock = ( collectionSlug, theme, index ) => {
+		return (
+			<ThemeBlock
+				collectionSlug={ collectionSlug }
+				getActionLabel={ this.getActionLabel }
+				getButtonOptions={ this.getOptions }
+				getPrice={ this.getPremiumThemePrice }
+				getScreenshotUrl={ this.getScreenshotUrl }
+				index={ index }
+				isActive={ this.isThemeActive }
```

Comment:
Eventually, we will not show the active theme in both collections and regular lists.

We can filter out the active theme from `themes` and remove this prop from the collection's items.